pkg/systemd/quadlet/quadlet.go

```diff
@@ -961,6 +963,22 @@ func ConvertKube(kube *parser.UnitFile, isUser bool) (*parser.UnitFile, error) {
 
 	addNetworks(kube, KubeGroup, service, execStart)
 
+	updateMaps := kube.LookupAllStrv(KubeGroup, KeyAutoUpdate)
+	for _, update := range updateMaps {
+		val := strings.SplitN(update, "/", 2)
+		switch {
```

Comment:
I'd prefer to just pass the values over without trying to validate them (i.e, compare to "registry" etc.).  Only `auto update ` should do that IMO to avoid scattering.

Reply:
@rhatdan @ygalblum this comment/issue is still open.  While I don't anticipate any new auto-update policies, I do think that scattered logic is imposing a risk when adding a new one.  In other words, IMO we should keep input _validation_ to a minimum in Quadlet.